code/modules/admin/admin.dm

```diff
@@ -900,6 +900,21 @@ var/global/noir = 0
 			else
 				alert("You need to be at least a Secondary Adminstrator to jump to mobs.")
 
+		if ("observe")
+			if(src.level >= LEVEL_SA)
+				var/mob/M = locate(href_list["target"])
+				if (!M) return
+				if (isobserver(M))
+					boutput(usr, "<span class='alert'>You can't observe a ghost.</span>")
```

Comment:
I might be totally wrong on whether or not it counts as "observing" but admins can click drag their ghost onto another ghost and observe it.

Players used to be able to do this too but it was patched as an exploit as they could observe admins in restricted locations